settings/development/PandoraSettings_Streaming_Neutrino_DUNEFD.xml

```diff
@@ -0,0 +1,640 @@
+    <algorithm type = "LArListDeletion">
+        <PfoListNames>ShowerParticles3D</PfoListNames>
+    </algorithm>
+    <algorithm type = "LArDLClusterCharacterisation">
+        <InputClusterListNames>TrackClustersU TrackClustersV TrackClustersW ShowerClustersU ShowerClustersV ShowerClustersW</InputClusterListNames>
+        <OverwriteExistingId>true</OverwriteExistingId>
+    </algorithm>
+
+    <algorithm type = "LArShowerGrowing">
+        <InputClusterListNames>ShowerClustersU ShowerClustersV ShowerClustersW</InputClusterListNames>
+    </algorithm>
+
+    <algorithm type = "LArThreeDShowers">
+        <InputClusterListNameU>ShowerClustersU</InputClusterListNameU>
+        <InputClusterListNameV>ShowerClustersV</InputClusterListNameV>
+        <InputClusterListNameW>ShowerClustersW</InputClusterListNameW>
+        <OutputPfoListName>ShowerParticles3D</OutputPfoListName>
+        <MinShowerMatchedPoints>15</MinShowerMatchedPoints>
+        <ShowerTools>
+            <tool type = "LArClearShowers"/>
+            <tool type = "LArSplitShowers"/>
+            <tool type = "LArSimpleShowers"/>
+        </ShowerTools>
+    </algorithm>
+
+    <!--2 view Calo matching for showers - U V -->
+    <algorithm type = "LArTwoViewTransverseTracks">
+        <InputClusterListName1>ShowerClustersU</InputClusterListName1>
+        <InputClusterListName2>ShowerClustersV</InputClusterListName2>
+        <OutputPfoListName>ShowerParticles3D</OutputPfoListName>
+        <LocalMatchingScoreThreshold>0.90</LocalMatchingScoreThreshold>
+        <DisplayPotentialMatches>false</DisplayPotentialMatches>
+        <MinSamples>13</MinSamples>
+        <TrackTools>
+            <tool type = "LArTwoViewThreeDKink"/>
+            <tool type = "LArTwoViewClearTracks">
+                <MinMatchingScore>0.95</MinMatchingScore>
+                <MinLocallyMatchedFraction>0.0</MinLocallyMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewClearTracks">
+                <MinMatchingScore>0.0</MinMatchingScore>
+                <MinLocallyMatchedFraction>0.30</MinLocallyMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewLongTracks">
+                <MinMatchingScore>0.0</MinMatchingScore>
+                <MinMatchedFraction>0.30</MinMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewLongTracks">
+                <MinMatchingScore>0.95</MinMatchingScore>
+                <MinMatchedFraction>0.0</MinMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewSimpleTracks">
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+        </TrackTools>
+    </algorithm>
+    <!--2 view Calo matching for showers - U W -->
+    <algorithm type = "LArTwoViewTransverseTracks">
```

Comment:
These are all-new, and not actually in the latest pre-DL neutrino file? (That's OK, of course, just obviously a question of alg choice and performance optimisation.)

Reply:
I think these will be part of Maria's updates for the new production.